new blog post - h2o usability study (#387)
harvard-lil/website-static: app/_posts/2022-09-23-h2o-usability-study-do-students-want-physical-casebooks.md

---
title: 'H2O usability study: do students want physical casebooks?'
author: catherine-brobston
---

> This summer, one of our RAs, Seonghee Lee, ran a study among current law students that is helping us reconsider some of the longstanding assumptions around student reading preferences and informing future development of our [H2O Open Casebook](https://opencasebook.org) platform. 

Reviewer: ```suggestion
This summer, one of our research assistants, Seonghee Lee, ran a study among current law students that is helping us reconsider some longstanding assumptions about student reading preferences and informing future development of the [H2O Open Casebook](https://opencasebook.org) platform. 

```

> H2O was launched in an early form in 2012, and for years we worked under the assumption that most books written with H2O would eventually be read in a print format. We have put a lot of work into improving the export experience so that professors can create a book using the H2O platform, export it as a Word doc, format it how they like, and distribute it to students as a low cost print-on-demand book or as a PDF they can print. Our expectations of reading formats began to evolve as we heard authors start to ask for multimedia options like video in their H2O casebooks, but we still heard strong feedback from many professors that they needed a print option for their students. 

Reviewer: ```suggestion
H2O was launched in an early form in 2012, and for years we worked under the assumption that most books written with H2O would eventually be read in a print format. We have put a lot of work into improving the export experience so that professors can create a book using the H2O platform, export it as a Word document, format it as they like, and distribute it to students as a low-cost, print-on-demand book or as a printable PDF. Our expectations of reading formats began to evolve as we heard authors start to ask for multimedia options like video in their H2O casebooks, but we still heard strong feedback from many professors that they needed a print option for their students. 

```

However, as with so many things, 2020 may have changed what we thought we knew by resetting students' expectations and preferences for their learning materials. This summer we spoke to 21 current law students who had not used H2O, and more than half told us that they prefer digital casebooks over physical texts. Cost was an obvious factor---if a digital book costs less than a physical book, they want the digital book---but many also cited their use of digital notetaking and writing tools as well as the clunkiness and inconvenience of heavy, printed books in their backpacks. Nine of those 21 students talked to us over Zoom (the others completed a survey), and once we were able to show those nine how H2O worked, they all said they could see themselves reading and annotating H2O directly on the platform. 

> While these conversations cut against the commonly held wisdom around student reading preference, they align with anecdotes I've been hearing from students. When chatting with some current students at a library event at Harvard Law School earlier this week, most told me that when a professor assigned an H2O book they read it on the H2O platform, even when a professor had created a print option for them. 

Reviewer: ```suggestion
While these conversations cut against the common wisdom about student reading preferences, they align with anecdotes I've been hearing from students. When chatting with some current students at a library event at Harvard Law School earlier this week, most told me that when a professor assigned an H2O book they read it on the H2O platform, even when the professor had created a print option. 

```

Of course, all of these conversations put together still add up to a small number of law students, and even if it is only a minority of students who prefer physical books, we want to make sure H2O is a platform that can meet those students' learning needs as well. We will continue to support professors who want to create printable versions of their H2O books for their students.

But these early conversations about how students prefer to read and to learn are forcing us to ask new questions, too. What tools and capabilities do we owe students who are reading H2O directly on the platform? How can we work with professors to better understand their students' reading preferences? What expertise can we learn from in designing a digital reading platform that is as effective (or better!) than physical reading? 

> Some early answers come directly from the usability study---students were most concerned with whether a digital reading platform like H2O has embedded annotation tools they can use to mark up the cases and inform the outlines they make for their classes. While many students thought they could use the annotation features already built into H2O, this feedback may point to a separate, student-centric set of annotation tools in H2O down the line. For now, we've added some improved UI to better direct readers of H2O casebooks to the annotation tools already there. 

Reviewer: ```suggestion
Some early answers come directly from the usability study---students were most concerned with whether a digital reading platform like H2O has embedded annotation tools they can use to mark up cases and inform the outlines they make for their classes. While many students thought they could use the annotation features already built into H2O, this feedback may point to a separate, student-centric set of annotation tools in H2O down the line. For now, we've added some improved UI to better direct readers of H2O casebooks to the annotation tools already there. 

```
